packages/plugin-kit/src/createSliceMachinePluginRunner.ts

```diff
@@ -127,14 +130,11 @@ export class SliceMachinePluginRunner {
 		let plugin: SliceMachinePlugin | undefined = undefined;
 
 		if (typeof resolve === "string") {
-			// Import plugin
-			try {
-				const raw = await import(resolve);
-				plugin = raw.default || raw;
-			} catch {
-				// If the plugin could not be imported, we will
-				// throw a custom error later.
-			}
+			// Import plugin as a file from the project
+			const raw = await createRequire(
+				path.resolve(this._project.root, "noop.js"),
+			)(resolve);
```

Comment:
Did you come across an issue with using `import()`? Can `import()` load CJS?

Reply:
The issue I had was that it was trying to load the adapter from this package rather than from the test project root. We might just want to use that as a fallback 🤔

Reply:
Oh right, `require()` lets you specify its resolution paths while `import()` doesn't (yet). I've been linking `@slicemachine/adapter-next` into `@slicemachine/plugin-kit` during development, but that's not a good solution.

Okay cool, let's leave this here. We may run into CJS vs ESM issues in the future since I assume this always loads CJS instead, but we can tackle that issue when the time comes.